frontend-v2/src/components/CourseCard.tsx

```diff
@@ -0,0 +1,85 @@
+interface CourseCardProps {
+  code: string;
+  name: string;
+  rating: number;
+  reviews: number;
+  terms: {
+    summer?: boolean;
+    term1?: boolean;
+    term2?: boolean;
+    term3?: boolean;
+  };
+}
+
+const CourseCard = ({
```

Comment:
lets try and use function going forward:
`export default function CourseCard()`